### ISSUE TITLE
Document configurability of configurable rules

### ISSUE THREAD

**Issue by fredpi** [realm/SwiftLint#2698]
I just looked up the repo for documentation of the `nesting` rule as I want to configure the nesting level, at which the `nesting` rule creates a warning.

Only after searching old issues I found out that what I'm looking for is actually possible:

```yml
nesting:
  type_level: 1
```

Maybe I oversaw something, but if that's not the case, shouldn't there be another way to find out about configurable rules other than browsing old issues / looking up the code directly? The `Rules.md` file could be the proper place for such a documentation.

**Comment by marcelofabri** [comment]
Duplicate of #1653.
